src/graphql/typeDefs/GDPR-User/mutation.js

```diff
@@ -0,0 +1,5 @@
+export default `
+type Mutation {
+  deleteGDPRUser(accountLinkId: String!): HTTPStatus! @isAuthenticatedOnQuery
```

Comment:
Sorry I do not have much context on this, but would this be used by users that are logged in to (requests scheduling of) deletion of their own account?

If this is the case, should we allow calling `deleteGDPRUser` but instead of using accountLinkedId from input use the ID we get from the users context?

Reply:
100% we should pull this off context. It was around this point I made a decision to get the PR ready for a merge and reduce scope creep. 

I'd happily remove it for now.

Reply:
👍